scripts/docker-run-usecases.sh

```diff
@@ -20,140 +20,160 @@ export DOCKER=1
     ./scripts/dartagnan.sh \
         -m power \
         -t ${solver} \
         -p reachability,liveness \
         -DALGORITHM=${QSPINLOCK_ALGORITHM} \
-        client-code.c | tee results/out10-dartagnan-${solver}-power-qspinlock-both-none.txt
+        client-code.c | tee results/out11-dartagnan-${solver}-power-qspinlock-both-none.txt
 
-    # 11    Verifying CNA under LKMM using Dartagnan,
+    # 12    Verifying CNA under LKMM v00 using Dartagnan,
     #       applying fixes 1, 2, 3, 4 and 5.
     #       Expect no violation found and result UNKNOWN.
     ./scripts/dartagnan.sh \
-        -m lkmm \
+        -m lkmm-v00 \
         -t ${solver} \
         -p reachability,liveness \
         -DALGORITHM=${CNA_ALGORITHM} \
         -DFIX1 -DFIX2 -DFIX3 -DFIX4 -DFIX5 \
-        client-code.c | tee results/out11-dartagnan-${solver}-lkmm-cna-both-12345.txt
+        client-code.c | tee results/out12-dartagnan-${solver}-lkmm_v00-cna-both-12345.txt
+
+    # 13    Verifying CNA under LKMM v01 using Dartagnan,
+    #       without applying any fix.
+    #       Expect no violation found and result UNKNOWN.
+    ./scripts/dartagnan.sh \
+        -m lkmm-v01 \
+        -t ${solver} \
+        -p reachability,liveness \
+        -DALGORITHM=${CNA_ALGORITHM} \
+        client-code.c | tee results/out13-dartagnan-${solver}-lkmm_v01-cna-both-none.txt
 
-    # 12    Verifying CNA on Armv8 using Dartagnan,
+    # 14    Verifying CNA on Armv8 using Dartagnan,
     #       without applying any fix.
     #       Expect no violation found and result UNKNOWN.
     ./scripts/dartagnan.sh \
         -m armv8 \
         -t ${solver} \
         -p reachability,liveness \
         -DALGORITHM=${CNA_ALGORITHM} \
-        client-code.c | tee results/out12-dartagnan-${solver}-armv8-cna-both-none.txt
+        client-code.c | tee results/out14-dartagnan-${solver}-armv8-cna-both-none.txt
 
-    # 13    Verifying CNA on RISC-V using Dartagnan,
+    # 15    Verifying CNA on RISC-V using Dartagnan,
     #       without applying any fix.
     #       Expect no violation found and result UNKNOWN.
     ./scripts/dartagnan.sh \
         -m riscv \
         -t ${solver} \
         -p reachability,liveness \
         -DALGORITHM=${CNA_ALGORITHM} \
-        client-code.c | tee results/out13-dartagnan-${solver}-riscv-cna-both-none.txt
+        client-code.c | tee results/out15-dartagnan-${solver}-riscv-cna-both-none.txt
 
-    # 14    Verifying qspinlock on Power using Dartagnan,
+    # 16    Verifying qspinlock on Power using Dartagnan,
     #       without applying any fix.
     #       Expect no violation found and result UNKNOWN.
     ./scripts/dartagnan.sh \
         -m power \
         -t ${solver} \
         -p reachability,liveness \
         -DALGORITHM=${CNA_ALGORITHM} \
-        client-code.c | tee results/out14-dartagnan-${solver}-power-cna-both-none.txt
+        client-code.c | tee results/out16-dartagnan-${solver}-power-cna-both-none.txt
 
-done
+done
```

Comment:
```suggestion
done

```

undo the removal of the CR